scripts/sample_make_output.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+TSHARK_EXECUTABLE="$1"
+SAMPLE_DIR="$2"
+TYPE="$3"
+REQ_VERSION="$4"
+
+${TSHARK_EXECUTABLE} --version > /dev/null 2> /dev/null
+if [ "$?" != "0" ]; then
+    echo "Executable for tshark doesn't exists (${TSHARK_EXECUTABLE})"
+    exit 0
+fi
+
+echo "Creating output '${TYPE}' for ${SAMPLE_DIR}:"
+FILE=`basename "${SAMPLE_DIR}"`
+
+TSHARK_VERSION=`${TSHARK_EXECUTABLE} --version | head -1 | cut -d' ' -f 3 | cut -d'.' -f1,2`
+if [ -n "${REQ_VERSION}" ]; then
+    if [ "${REQ_VERSION}" != "${TSHARK_VERSION}" ]; then
+        echo "  FAILED, required tshark version do not match running version"
+        exit 1
+    fi
+fi
+
+cd "${SAMPLE_DIR}"
+
+if [ -f "${FILE}.pcap.gz" ]; then
+        FILE_PCAP="${FILE}.pcap.gz"
+elif [ -f "${FILE}.pcapng.gz" ]; then
+        FILE_PCAP="${FILE}.pcapng.gz"
```

Comment:
There are capture formats other than (compressed) pcap like android logcat, etc. What do you think about using a single extension (like FOO.pcap or FOO.cap) even if it is compressed? Otherwise we might have a lot of files here.

Alternatively, we can stick to the original convention of looking for FOO given FOO.pdml (e.g. dns.pcapng.pdml)